packages/docs/components/SideNav/SideNavGroup/SideNavGroup.tsx

```diff
@@ -1,19 +1,13 @@
 import { Box, H4 } from '@bigcommerce/big-design';
 import React from 'react';
-import styled from 'styled-components';
 
 import { List } from '../../List';
 
-const StyledList = styled(List)`
-  margin: ${({ theme }) => theme.spacing.none};
-  padding: ${({ theme }) => theme.spacing.none};
-`;
-
 export const SideNavGroup: React.FC<{ title: string }> = (props) => {
   return (
     <Box marginTop={{ mobile: 'xxSmall', tablet: 'xLarge' }} marginHorizontal={{ mobile: 'medium', tablet: 'none' }}>
       <H4>{props.title}</H4>
-      <StyledList bulleted={false}>{props.children}</StyledList>
+      <List reset>{props.children}</List>
```

Comment:
another optional prop name could be `resetListStyle` , `resetStyle`  so it could be more descriptive 🍹